webapp/src/components/add_issue/add_issue.jsx

```diff
@@ -0,0 +1,328 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    makeStyleFromTheme,
+    changeOpacity,
+} from 'mattermost-redux/utils/theme_utils';
+
+import TextareaAutosize from 'react-textarea-autosize';
+
+import FullScreenModal from '../modals/modals.jsx';
+import Button from '../../widget/buttons/button';
+import Chip from '../../widget/chip/chip';
+import AutocompleteSelector from '../user_selector/autocomplete_selector.tsx';
+import './add_issue.scss';
+import CompassIcon from '../icons/compassIcons';
+import {getProfilePicture} from '../../utils';
+
+const PostUtils = window.PostUtils;
+
+export default class AddIssue extends React.PureComponent {
+    static propTypes = {
+        visible: PropTypes.bool.isRequired,
+        message: PropTypes.string.isRequired,
+        postID: PropTypes.string.isRequired,
+        assignee: PropTypes.object,
+        closeAddBox: PropTypes.func.isRequired,
+        submit: PropTypes.func.isRequired,
+        theme: PropTypes.object.isRequired,
+        autocompleteUsers: PropTypes.func.isRequired,
+        openAssigneeModal: PropTypes.func.isRequired,
+        removeAssignee: PropTypes.func.isRequired,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            message: '',
```

Comment:
Not sure if this messes with the flow, but it seems we should be initializing like this

```suggestion
            message: props.message || '',
```